plexpy/pmsconnect.py

```diff
@@ -692,11 +697,225 @@ def get_metadata_children_details(self, rating_key=''):
+
+
+        Output: List for dicts
+
+        # Adding all_params=1 Makes the call insane slow.
+        """
+        # Add a cache?
+
+        use_watched_older_then_sort = True
+        if watched_older_then is None:
+            use_watched_older_then_sort = False
+            watched_older_then = time.time()
+        else:
+            watched_older_then = int(watched_older_then)
+
+        if plexpy.CONFIG.PMS_URL:
+            url_parsed = urlparse(plexpy.CONFIG.PMS_URL)
+            hostname = url_parsed.hostname
+            port = url_parsed.port
+            self.protocol = url_parsed.scheme
+        else:
+            hostname = plexpy.CONFIG.PMS_IP
+            port = plexpy.CONFIG.PMS_PORT
+            self.protocol = 'http'
+
+        b_url = '%s://%s:%s' % (self.protocol, hostname, port) 
+
+        h = {'Accept': 'application/json',
+             'X-Plex-Token': plexpy.CONFIG.PMS_TOKEN
+            }
+
+        hide_watched = 'unwatched' if hide_watched else 'all'
+
+        def fetch(s=''):
+            result = request_json(b_url + s, headers=h)
+            if result:
+                return result
+
+        def maybe_number(v):
+            try:
+                v = int(v)
+            except:
+                try:
+                    v = float(v)
+                except:
+                    pass
+            return v
+
+        result = fetch('/library/sections/all').get('_children')
+
+        # Final results for data grab
+        f_result = []
+
+        # List of items and urls passed to req.f
+        files_urls = []
+
+        # dicts from req.f is stored here
+        futures_results = []
+
+        # Start fetching data
+        if result:
+            for sections in result:
+                if ignore_section != sections['title']:
+                    section = fetch('/library/sections/%s/%s' % (sections['key'], hide_watched))
+                    for item in section['_children']:
+                        
+                        d = {'title': item.get('title'),
+                            'type': item['type'],
+                            'ratingKey': item['ratingKey'],
+                            'updatedAt': item.get('updatedAt', 0),
+                            'addedAt': item.get('addedAt', 0),
+                            'viewCount': item.get('viewCount', 0),
+                            'files': [],
+                            'lastViewedAt': item.get('lastViewedAt', 0),
+                            'viewOffset': item.get('viewOffset', 0),
+                            'spaceUsed': 0, # custom 
+                            'isSeen': 0, # custom
+                            '_elementType': item['_elementType']
+                        }
+
+                        # Only movies has the files listed here
+                        if item['_elementType'] == 'Video':
+                            d['viewCount'] = item.get('viewCount', 0)
```

Comment:
Same as above, view count is only for yourself, not over all users.

Total views should be pulled from the PlexPy db.